pyrisco/local/risco_socket.py

```diff
@@ -33,7 +33,10 @@ async def connect(self):
       self._queue = asyncio.Queue()
       self._listen_task = asyncio.create_task(self._listen())
       self._crypt = RiscoCrypt(self._encoding)
-      panel_id = int(await self.send_result_command('RID'))
+      try:
+        panel_id = int(await self.send_result_command('RID'))
+      except ValueError:
+        panel_id = int(await self.send_result_command('RID'), 16)
```

Comment:
```suggestion
      panel_id = int(await self.send_result_command('RID'), 16)
```
I think it's always hexadecimal

Reply:
Okay, I will change it. I did it in a try/catch block to not break systems that work with this version